x-pack/plugins/ingest_manager/server/routes/epm/index.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { IRouter } from 'kibana/server';
+import { PLUGIN_ID, EPM_API_ROUTES } from '../../constants';
+
+export const registerRoutes = (router: IRouter) => {
+  router.get(
+    {
+      path: EPM_API_ROUTES.CATEGORIES_PATTERN,
+      validate: false,
+      options: { tags: [`access:${PLUGIN_ID}`] },
+    },
+    async (context, req, res) => {
+      return res.ok({ body: { hello: 'world' } });
+    }
+  );
+
+  router.get(
+    {
+      path: EPM_API_ROUTES.LIST_PATTERN,
+      validate: false,
+      options: { tags: [`access:${PLUGIN_ID}`] },
+    },
+    async (context, req, res) => {
+      return res.ok({ body: { hello: 'world' } });
+    }
+  );
+
+  router.get(
+    {
+      path: `${EPM_API_ROUTES.INFO_PATTERN}/{filePath*}`,
+      validate: false,
+      options: { tags: [`access:${PLUGIN_ID}`] },
+    },
+    async (context, req, res) => {
+      return res.ok({ body: { hello: 'world' } });
+    }
+  );
+
+  router.get(
+    {
+      path: EPM_API_ROUTES.INFO_PATTERN,
+      validate: false,
+      options: { tags: [`access:${PLUGIN_ID}`] },
+    },
+    async (context, req, res) => {
+      return res.ok({ body: { hello: 'world' } });
+    }
+  );
+
+  router.get(
```

Comment:
I think this one should be POST, so that users can't accidentally install a package by exploring the API in the browser or with `curl`. I wanted to propose this change for a long time but didn't get around to doing it, but I think this would be the right time to do it.

Reply:
👍 I will leave adjustment of these routes up to whoever does the EPM migration work as the EPM stuff here is boilerplate